addons/diagnostic/test_assertions.sqf

```diff
@@ -13,14 +13,14 @@ private ["_a", "_b"];
 ERROR("Testing that ERROR() is output correctly");
 _a = false;
 
-ASSERT_TRUE(_a,"Checking ASSERT_TRUE(" + str(_a) + ") outputs an error correctly");
+ASSERT_TRUE(_a,"Checking AXSSERT_TRUE(" + str(_a) + ") outputs an error correctly");
```

Comment:
```
error: Preprocessor error: Function call with incorrect number of arguments, expected `2` got `1`. `Token { symbol: Word("ASSERT_TRUE"), source: Position { start: (376, (16, 26)), end: (387, (16, 37)), path: "/addons/diagnostic/test_assertions.sqf" }, parent: None }`

   --> addons/diagnostic/test_assertions.sqf:16:26
    |
 16 | ASSERT_TRUE(_a,"Checking ASSERT_TRUE(" + str(_a) + ") outputs an error correctly");
    |                          ^ Expects 2 arguments
```
I think this is a false error, the inner `ASSERT_TRUE` is inside `"` so it shouldn't be expanded.

Reply:
this is how the game demacros it
```
if (not (_a)) then     {         ['cba', 'diagnostic', "Assertion failed!", 'Assertion (_a) failed!\n\n' + ("Checking ASSERT_TRUE(" + str(_a) + ") outputs an error correctly"), "x\cba\addons\diagnostic\test_assertions.sqf", 15 + 1] call CBA_fnc_error;     };
```

Reply:
Right, so only `'` should be expanded, not `"`, correct?